packages/utils/src/address.ts

```diff
@@ -3,11 +3,14 @@ import { privateToAddress, toChecksumAddress } from 'ethereumjs-util'
 export type Address = string
 
 export function eqAddress(a: Address, b: Address) {
-  return a.replace('0x', '').toLowerCase() === b.replace('0x', '').toLowerCase()
+  return trimLeading0x(a).toLowerCase() === trimLeading0x(b).toLowerCase()
 }
 
+export function trimLeading0x(input: string) {
```

Comment:
This is a duplicate of this: https://github.com/celo-org/celo-monorepo/blob/master/packages/utils/src/signatureUtils.ts#L147

Though this is actually easier to find, so please migrate the of the other one over to this. Ditto for `eqAddress ` above and `areAddressesEqual` in sigUtils